src/web/assets/updater/src/Updater.js

```diff
@@ -77,7 +75,7 @@ import './update.scss';
             }
 
             if (state.nextAction) {
-                this.postActionRequest(state.nextAction);
+                this.sendActionRequest('POST', state.nextAction);
```

Comment:
There is no internal `sendActionRequest` method. Do we want to use the internal postActionRequest method or